edx_django_utils/__init__.py

```diff
@@ -2,7 +2,7 @@
 EdX utilities for Django Application development..
 """
 
-__version__ = "3.14.0"
+__version__ = "3.14.1"
```

Comment:
FYI: See https://open-edx-proposals.readthedocs.io/en/latest/oep-0047-bp-semantic-versioning.html to understand a better version update.

Reply:
Thank you for sharing this document. Now updated version accordingly.